google_drive/tests/integration/xml/image.xml

```diff
@@ -1 +1 @@
-<google-document embed_code="&lt;img src='https://docs.google.com/drawings/d/1lmmxboBM5c_0WCTjhAxBdkpqQb3T8VSwtuG0TRR1ODQ/pub?w=960&amp;h=720'&gt;"/>
+<google-document embed_code="&lt;img src='https://docs.google.com/drawings/d/e/2PACX-1vR5IEdW9QtEDQG8Kr0ZyigIVJNE8PTpet3H1AcLH0LsCBIVlN-onsTnmP5cfD44cwzrAS3jacvPP6q6/pub?w=960&amp;h=720'&gt;"/>
```

Comment:
Where is this image being hosted? In the edx google account or somewhere else?  Who can change it?  Can we add some context in a comment if we need to replace this again?

Reply:
@feanil This image is hosted with my edX google account, can it cause these tests failing again if my account deleted in future? If yes can you provide a link for a image which will not change?

Reply:
@waheedahmed yea if this image is currently hosted in your account then this can cause an issue.  Does this image need to be hosted on google for the test to work correctly?

Reply:
@feanil yes, just provide me the google document embed code for any image hosted on edX google, I'll update.

Reply:
@feanil Can I merge this or are you going to provide an image URL?

Reply:
@waheedahmed I think putting this in your personal drive will cause it to get deleted at some point in the future. We should put it in a team drive.  I think thing thing to do is as follows:

1. Merge the PR as-is
2. Get a new drive created by IT to host this image.
3. Update the xblock to use the image from the new drive.

Reply:
Done.